src/css/components/po-table/po-table-column-manager/po-table-column-manager.css

```
@@ -91,14 +91,49 @@ po-table-column-manager .po-popover .po-popover-content {
   outline: dashed 2px var(--color-table-column-manager-outline-color-focus);
 }
 
+.container-arrows-columns-manager {
```

```suggestion
.po-container-arrows-columns-manager {
```
Aplicar este padrão para as demais classes css criadas